ui/blockly/blocks/clova.js

```diff
@@ -102,5 +102,14 @@ Blockly.defineBlocksWithJsonArray([  // BEGIN JSON EXTRACT
 		"colour": 330,
 		"tooltip": "",
 		"helpUrl": ""
+	},
+	{
+		"type": "clova_end_session",
+		"message0": "end session",
+		"previousStatement": null,
+		"nextStatement": null,
+		"colour": 230,
```

Comment:
The colour should be 330 as the same as the session attribute.

Reply:
Fixed at https://github.com/umakoz/blova/pull/5/commits/58fabefab18d76effddcac5e0f5bfb0004fef119